Formula/util-linux.rb

```diff
@@ -20,14 +20,17 @@ class UtilLinux < Formula
     sha256 mojave:        "de7bfed47b70d497e2406b7813b966aad7a0436e6fd129d4e12f5df5757e3ef9"
   end
 
-  keg_only "macOS provides the uuid.h header"
+  on_macos do
+    keg_only "macOS provides the uuid.h header"
+  end
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "pkg-config" => :build
   depends_on "gettext"
 
+  uses_from_macos "bison" => :build
```

Comment:
Do we need `bison` to call `./autogen.sh` only? If so, could you please add it to a patch comment below (to remove this dependency in the next release)

Reply:
I can't move it because our style audit. I added it because the build complained that it needed bison.